code/style.css

```diff
@@ -2,6 +2,298 @@
+  padding-top: 30px;
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.15);
+  font-size: 15px;
+}
+    
+.form-group {
+  display: flex;
+  justify-content: center;   
+}
+  
+.form-group label {
+  justify-content: center;
+  padding:10px;
+  border-radius:10px;
+}
+  
+.form-group-button {
+  display: flex;
+  justify-content: center;
+}
+  
+h3 {
+  display: flex;
+  justify-content: center;
+}
+  
+button {
+  height: 50px;
+  background-color: white;
+  border-radius: 15px;
+  border-style: hidden;
+  font-size: 20px;
+  Font-Family: 'Playfair Display', Serif;
+  box-shadow: grey 2px 2px;
+}
+  
+input {
+  font-size: 20px;
+}
+  
+  /*Color Validations */ 
+  
+input:valid {
+  background-color: #63C380;
+}
+  
+input:invalid {
+  border-color:#FE6C6C ;
+}
+  
+label {
+  display: block;
+  margin-bottom: 30px;
+}
+  
+  
+   /* Accordion */
+  
+.answer {
+  display: none;
+}
+  
+   /* Accordion  box styling and placement*/
+  
+.accordion {
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  margin-left: 5px;
+  margin-top: 100px;
+  margin-bottom:90px ;
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.15);
+}
+  
+.accordion h1 {
+  display: flex;
+  text-align: center;
+  font-size: 30px;
+  background-color:rgb(250, 240, 230);;
+}
+  
+.answer {
+  height: 0px;
+  opacity: 0;
+  overflow: hidden;
+  transition: all 1s ease-in-out;
+  -webkit-transition: all 1s ease-in-out;
+  -moz-transition: all 1s ease-in-out;
+  -ms-transition: all 1s ease-in-out;
+  -o-transition: all 1s ease-in-out;
+}
+  
 .active + .answer {
-  /* some styling here */
+  display: inline-block;
+  opacity: 1;
+  height: 80px;
+  background-color: #FAEBD7;
+}
+  
+.question {
+  font-size: 20px;
+  margin:12px;
+  padding:10px;
+  cursor: pointer;
+  border-bottom: 2px solid white;
+}
+  
+  
+  /* the icon in the accordion    */
+  
+.question::after {
+  content: "\002B";
+  font-size: 25px;
+  float: left;
+  margin-right: 10px;
+}
+  
+.active::after {
+  content: "\002D";
+}
+  
+.question:hover {
+  background-color:#FAEBD7;
+}
+  
+  /*Footer*/ 
+  
+footer{
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  background-color: white; 
+}
+  
+footer p {
+  text-align: center;
+  font-size: 18px;
+  color:black;
 }
+  
+footer img {
+  width: 50px;
+  margin-bottom: 10px;
+}
+
+
+  
+  /***  Media Quaries  Tablet  ***/
+  
+@media screen and (min-width: 667px) and (max-width:1024px) {
+  .hero-img {
+    height: 700px;
+  }
+
+  .hero-img h1 {
+    font-size: 60px;
+  }
+
+  .weare{
+    font-size: 20px;
+  } 
+  .sign-up-container{
+    justify-content: left;
+    margin-left: 80px;
+  }
+  .sign-up {
+    font-size: 12px;
+    justify-content: left;
+    margin-left: 15px;
+  }
+  .container {
+    display: inline-flex;
+    margin-left: 10px;
+    flex-direction: row;
+  }
+  form {
+    margin-left: 20px;
+    height:570px ;
+  }
+  .accordion {
```

Comment:
Try to be consistent with the code structure, this needs some air in between to be more readable